<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferWindow</class>
 <widget class="QWidget" name="TransferWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="wgtToolBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>26</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="toolBarLayout">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="lmcTransferListView" name="lvTransferList">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="bottomLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear List</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>lmcTransferListView</class>
   <extends>QListView</extends>
   <header>transferlistview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lvTransferList</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>TransferWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
